src/Maestro/maestro-angular/src/styles.scss

```diff
@@ -0,0 +1,20 @@
+/* You can add global styles to this file, and also import other style files */
+
+@import "~bootstrap/scss/bootstrap";
```

Comment:
Does angular do CSS shaking?  We use a very, very small subset of bootstrap, so we could have a much smaller footprint if we only included the like 4 styles we actually care about (we don't have any forms, for example, which are a big pile of CSS)